.github/workflows/build.yml

```diff
@@ -79,7 +97,6 @@ jobs:
         with:
           arguments: --refresh-dependencies --stacktrace --scan clean build -x spotlessCheck
```

Comment:
Wonder if we need/want `spotlessCheck` here and basically everywhere, as we have a dedicated `code-format-check` job?

Reply:
[It's actually excluded with the `-x` option...](https://docs.gradle.org/current/userguide/command_line_interface.html#sec:excluding_tasks_from_the_command_line) But I thought the same thing first. 😅

Reply:
Another example of "I don't know shit about Gradle"… 😅